app/src/main/java/com/github/braillesystems/learnbraille/ui/screens/browser/BrowserFragment.kt

```diff
@@ -58,6 +58,7 @@ class BrowserFragment : AbstractFragmentWithHelp(R.string.browser_help) {
                         application.showSymbolPrintRules[item.data.char].toString(),
                         item.data.brailleDots.spelling
                     )
+                    else -> TODO()
```

Comment:
It will throw `NotImplementedError`, maybe add a text instead (e. g. "Undefined flying object")?

Reply:
(Even after implementing it for new symbol types)

Reply:
Oh, just forgot. Fixing...

Reply:
Some more TODOs there...

Reply:
You‘d better sleep well 🛌🏼 💤

Reply:
Of course, but TODOs are there for you ;)

Reply:
OK, I will work on them today, thanks

Reply:
Done